odoo/modules/module.py

```diff
@@ -314,8 +314,12 @@ def load_manifest(module, mod_path=None):
     try:
         manifest['version'] = adapt_version(manifest['version'])
     except ValueError as e:
-        if manifest.get("installable", True):
+        if manifest['installable']:
```

Comment:
This will make the key `installable` being required, which it's not desired 🤔

Reply:
It will not because `manifest` is an updated deep-copy of the default values: "installable" is always defined here even if it's not in the manifest.